markdown/docs/tools/cli/metrics_collection.md

```diff
@@ -1,3 +1,8 @@
+---
+title: 'Metrics Collection'
+weight: 20
+---
+
```

Comment:
I think this part is handled in the workflow of cli to update the docs in the website.

Reply:
This was creating an error for us in rendering docs